lib/private/DB/QueryBuilder/QueryBuilder.php

```diff
@@ -1096,7 +1096,7 @@ public function orHaving(...$having) {
 	 * Specifies an ordering for the query results.
 	 * Replaces any previously specified orderings, if any.
 	 *
-	 * @param string $sort The ordering expression.
+	 * @param string|IQueryFunction $sort The ordering expression.
```

Comment:
lets use
```suggestion
	 * @param string|ILiteral|IParameter|IQueryFunction $sort The ordering expression.
```

as on `addOrderBy`

Reply:
done